src/main/histFileMod.F90

```diff
@@ -84,75 +82,86 @@ module histFileMod
   character(len=max_namlen), public :: &
        hist_type1d_pertape(max_tapes)  = (/(' ',ni=1,max_tapes)/)   ! namelist: per tape type1d
 
-  character(len=max_namlen+2), public :: &
-       fincl(max_flds,max_tapes)         ! namelist-equivalence list of fields to add
+  logical, public :: &
+       hist_empty_htapes  = .false.      ! namelist: disable default-active history fields (which
+                                         ! only exist on history tape 1). Overridden by hist_fincl1 
+                                         ! flag.
```

Comment:
Hmmm, my first reaction here is that it isn't "overridden" by the hist_fincl1 flag. But, in a way it is in that anything in hist_fincl1 -- will be output. The way I've thought about it is that this turns all the default fields off -- and you have to explicitly add the list that you want to hist_fincl1.

Reply:
Makes sense, I reworded to " Use hist_fincl1 to enable select fields on top of this."